pkgs/by-name/ki/kindletool/package.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, pkg-config
+, libarchive
+, zlib
+, nettle
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "KindleTool";
+  version = "1.6.5";
+
+  src = fetchFromGitHub {
+    owner = "NiLuJe";
+    repo = "KindleTool";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-Io+tfwgRAPEx+TQKZLBGrrHGAVS6ndgOOh+KlBh4t2U=";
+  };
+
+  patchPhase = ''
+    substituteInPlace KindleTool/Makefile \
+      --replace "DESTDIR?=/usr/local" "DESTDIR?=$out"
```

Comment:
Can't we set this somehow without using substitute? Maybe `env.DESTDIR` or in `cmakeFlags`?

Besides that, I think this is good to go.

Reply:
Tried but not work.